README.md

```diff
@@ -62,6 +62,68 @@ $ make migrate-version
 1. Add migration to update plan table with gear enabled column
 2. Update `Plan` struct and `CanAccessGear` func in `pkg/gateway/model/plan.go`
 
+## Gear DB migration
+
+The following part is about gear db migration.
+
+If you come from skygear-server 0.x to 1.x, the biggest difference is that skygear next would not offer JIT db migration in most gears.
```

Comment:
It might be a bit misleading: It is because we don't expect/recommend gears developed with the SkyRecord API, so JIT DB migration is not offered.

The current statement makes it sounds like JIT DB Migration is another feature.